packages/contracts/test/attestation.test.ts

```diff
@@ -146,12 +146,10 @@ describe('Attestations', function () {
         const epochKey = BigInt(24910)
```

Comment:
We need one more test case

- Attest to a replacement field and check that the data matches the expected value

Check this in the emitted event for the attestation transaction

Reply:
Added new test case to attest with different values and verify that it is the expected value